### PR TITLE
add mobile view to index page

### PR DESCRIPTION
**Pull request by Arashfa0301**
add mobile view and a hamburger menu to index page

### CODE REVIEW

File: components/honoraryMember.tsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const HonoraryMember = ({ member }: { member: MemberProps }) => {
         {!isMobile && displayRole}
         <p />
       </Stack>
-      <div className={styles.verticleLine} />
+      <hr className={styles.verticleLine} />
```
Comment (danielyanghansen) [inline components/honoraryMember.tsx:73]: Fix spelling?
verticleLine -> verticalLine

File: styles/Member.module.css [hunk 2]
```diff
@@ -3,8 +3,10 @@
 }
 
 .verticleLine {
```
Comment (danielyanghansen) [inline styles/Member.module.css:5]: Fix spelling here too?

**Review by danielyanghansen — COMMENTED** [review]
Mobile view navbar functionality doesn't seem to work in vercel, but works fine from localhost server. Unsure of why. Might want to figure this out before merging.
Locally run version: LGTM :)

File: components/honoraryMember.tsx (hunk 1 above)
Comment (Arashfa0301) [reply components/honoraryMember.tsx]: thanks for noticing. I recall vercel not working properly at a previous occasion but after merging it didn't seem to have any effect. I'm not sure about what the problem could be but I'm open to suggestions.

**Review by ivarnakken — APPROVED** [review]
LGTM 💯 It's not perfect, but still nice and should be merged 😄

**Pull request merged**